### PR TITLE
Release notes for 1.5 Beta 3

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
Getting ready for next beta release.

 - [x] approve release notes
 - [x] ship to beta

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "1.5.0-beta3": [
+      "[???] Add support for reporting a binary file is in a conflicted state - #6072",
```
Comment (shiftkey) [inline changelog.json]: This is internal changes and not user-facing currently :fire:

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+      "[???] Handle really long banner messages - #6050",
+      "[Fixed] Ensure status after aborting a merge reflects reality - #6052",
```
Comment (shiftkey) [inline changelog.json]: The three fixes here are part of the Conflict Resolution flow that users noticed on beta, but we should omit it from the production release notes :fire:

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "1.5.0-beta3": [
+      "[???] Add support for reporting a binary file is in a conflicted state - #6072",
+      "[???] Handle really long banner messages - #6050",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Improved] Merge success banner handles very long branch names - #6050",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "1.5.0-beta3": [
+      "[Added] Will now reportup when a binary file is in a conflicted state - #6072",
```
Comment (shiftkey) [inline changelog.json]: This is internal changes and not user-facing currently 🔥

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] Handle really long banner messages - #6050",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Improved] Merge success banner handles very long branch names - #6050",
```

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] Changes are updated after aborting a merge - #6052",
```
Comment (shiftkey) [inline changelog.json]: :art: the three fixes here are part of the Conflict Resolution flow that users noticed on beta, but we should omit it from the production release notes :fire:

Comment (iAmWillShepherd) [reply changelog.json]: Just to clarify, are these fine for the beta release notes? Or should I flame them now?

Comment (iAmWillShepherd) [reply changelog.json]: 🔥

**Comment by iAmWillShepherd** [comment]
🍎

File: changelog.json [hunk 7]
```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "1.5.0-beta3": [
+      "[Fixed] Kill off unnecessary margin inside compare tab branch list - #6058",
```
Comment (shiftkey) [inline changelog.json]: This entry and the one below were also related to a regression that hasn't hit production yet. Let's omit them (yeah, this means it's an empty changelog now but that's okay).

**Comment by nerdneha** [comment]
Uhhh can't figure out how to undo my approval of changes. Happy to defer to @shiftkey's comment https://github.com/desktop/desktop/pull/6083#discussion_r230437062
```
This entry and the one below were also related to a regression that hasn't hit production yet - b8a6009. Let's omit them (yeah, this means it's an empty changelog now but that's okay).
```

**Pull request merged**